lib/PicoDb/Driver/Mssql.php

```diff
@@ -135,7 +135,7 @@ public function getLastId()
      */
     public function getSchemaVersion()
     {
-        $this->pdo->exec("IF (SELECT [object_id] FROM [sys].[tables] WHERE [object_id] = OBJECT_ID('".$this->schemaTable."')) IS NULL CREATE TABLE [".$this->schemaTable."] ([version] INT DEFAULT '0')");
+        $this->pdo->exec("IF (OBJECT_ID('".$this->schemaTable."')) IS NULL CREATE TABLE [".$this->schemaTable."] ([version] INT DEFAULT '0')");
```

Comment:
@Mr-Kaos are the parens around `OBJECT_ID(...)` necessary? Other examples I've seen of this approach omit them, e.g.

```
IF OBJECT_ID("table_name") IS NULL ...
```

Reply:
The parentheses are not required around `OBJECT_ID(...)`. I had only included them to make the contents of the `IF` statement clear.